openapi-generator-annotations/lib/src/openapi_generator_annotations_base.dart

```diff
@@ -337,17 +337,5 @@ enum Generator {
   ///
   /// You can read more about it here https://github.com/dart-ogurets/dart-openapi-maven
   dioAlt,
-
-  /// This uses the next gen dio generator. This is experimental and use at your own risk. It might be removed or renamed in the future
-  ///
-  /// You can read more about it here https://github.com/OpenAPITools/openapi-generator/pull/8869
-  dioNext,
-
-  /// This generates code based on the jaguar package Source gen is required
-  /// after generating code with this generator
-  /// corresponds to dart-jaguar
-  ///
-  /// An Http Api generator inspired by Retrofit for Dart
-  jaguar,
```

Comment:
Removing these causes:

```console
[WARNING] ../../../.pub-cache/git/openapi-generator-dart-fork-e2850af0da9618228ab03e37de1026190e43794b/openapi-generator/lib/src/openapi_generator_runner.dart:134:33: Error: Member not found: 'dioNext'.
          case annots.Generator.dioNext:
                                ^^^^^^^
../../../.pub-cache/git/openapi-generator-dart-fork-e2850af0da9618228ab03e37de1026190e43794b/openapi-generator/lib/src/openapi_generator_runner.dart:135:33: Error: Member not found: 'jaguar'.
          case annots.Generator.jaguar:
                                ^^^^^^
../../../.pub-cache/git/openapi-generator-dart-fork-e2850af0da9618228ab03e37de1026190e43794b/openapi-generator/lib/src/openapi_generator_runner.dart:229:29: Error: Member not found: 'dioNext'.
      case annots.Generator.dioNext:
                            ^^^^^^^
../../../.pub-cache/git/openapi-generator-dart-fork-e2850af0da9618228ab03e37de1026190e43794b/openapi-generator/lib/src/openapi_generator_runner.dart:235:29: Error: Member not found: 'jaguar'.
      case annots.Generator.jaguar:
                            ^^^^^^
```

Reply:
Made a PR on your fork: https://github.com/atoka93/openapi-generator-dart-fork/pull/1

Reply:
Thank you!